source/kraken/apply_disruption.cpp

```diff
@@ -98,6 +101,24 @@ nt::VehicleJourney* create_vj_from_old_vj(nt::MetaVehicleJourney* mvj,
     return new_vj;
 }
 
+std::string concatenate_impact_uris(const nt::MetaVehicleJourney& mvj) {
```

Comment:
this function is not used anymore right ? Maybe remove it then ?

Reply:
I thought I had removed it... :open_mouth: